src/main/java/com/github/manolo8/darkbot/backpage/NovaManager.java

```diff
@@ -0,0 +1,205 @@
+     * @throws IOException
+     */
+
+    private void updateResource() throws IOException {
+        String response = backpageManager.getConnection("ajax/captain.php", Method.POST)
+                .setRawParam("command", "getResources")
+                .getContent();
+
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class); //Converts the json string to JsonElement without POJO
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
+            this.data.setResourceAmount(jsonObj.get("item").getAsJsonObject().get("amount").getAsInt());
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+    }
+
+    /**
+     * Method to get list of Nova Agent/Captain
+     * @throws IOException
+     */
+
+    private void updateRosterList() throws IOException {
+        String response = backpageManager.getConnection("ajax/captain.php", Method.POST)
+                .setRawParam("command", "getRosterList")
+                .getContent();
+
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class); //Converts the json string to JsonElement without POJO
+
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
+            String resp = jsonObj.getAsJsonArray("rosterList").toString();
+            Agent[] rosterList = g.fromJson(resp, Agent[].class);
+
+            for (Agent agent : rosterList) {
+                this.data.addAgent(agent);
+            }
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+    }
+
+
+    /**
+     * Method to dismiss Nova Agent/Captain
+     * @param id
+     * @return
+     * @throws IOException
+     */
+
+    public boolean dismissAgent(int id) throws IOException {
+        String response = backpageManager.getConnection("ajax/captain.php", Method.POST)
+                .setRawParam("command", "dismissCaptain")
+                .setRawParam("captainId", id)
+                .getContent();
+        this.update(0);
+        JsonObject jsonObj = g.fromJson(response, JsonObject.class); //Converts the json string to JsonElement without POJO
+        if (jsonObj.get("result").getAsString().equalsIgnoreCase("OK")) {
+            return true;
+        } else {
+            System.out.println("NovaManager: " + response);
+        }
+        return false;
+    }
+
+    /**
+     * Method to buy Nova Agent/Captain
+     * @param amount
+     * @return
+     * @throws IOException
+     */
+
+    public boolean buyNova(int amount) throws IOException {
+        String response = backpageManager.getConnection("ajax/shop.php", Method.POST)
+                .setRawParam("action", "purchase")
+                .setRawParam("category", "special")
+                .setRawParam("itemId", "captain_captain-generic")
+                .setRawParam("amount", amount)
+                .setRawParam("level", "")
+                .setRawParam("selectedName", "")
+                .getContent();
```

Comment:
use setParam over setRawParam unless you know what you're doing.

setParam will convert things like spaces into the url-encoded versions, which is what you'd want if you had spaces